TauAnalysis/MCEmbeddingTools/python/EmbeddingPythia8Hadronizer_cfi.py

```diff
@@ -19,7 +19,8 @@
             MuHadCut = cms.string('Mu.Pt > 18 && Had.Pt > 25 && Mu.Eta < 2.1'),
             MuMuCut = cms.string('Mu1.Pt > 17 && Mu2.Pt > 8'),
             Final_States = cms.vstring('ElEl','ElHad','ElMu','HadHad','MuHad','MuMu'),
-            BosonPDGID = cms.int32(23)
+            BosonPDGID = cms.int32(23),
+            IncludeDY = cms.untracked.bool(True)
```

Comment:
Are you sure that you want to set `True` as default value?

Reply:
Ah yes, you are right. I was testing it `True` as well and forgot to update.